src/commonMain/kotlin/exchange.dydx.abacus/output/PerpetualState.kt

```diff
@@ -33,6 +33,7 @@ data class PerpetualState(
     val input: Input?,
     val availableSubaccountNumbers: IList<Int>,
     val transferStatuses: IMap<String, TransferStatus>?,
+    val trackStatuses: IMap<String, Boolean>?,
```

Comment:
does this need to be nullable? does empty work here?

Reply:
hmm... maybe. right now the abacus pattern is to return null for invalid data but we can probably change the default value to an empty map. i'll attempt to do it and timebox it for ~30 mins

Reply:
ah - everything is pulled from the typeless `data` map so everything **has** to be nullable until we get state schematized